Sources/ReadingListKit/ReadingListCell.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import DesignKit
+
+public struct ReadingListCell: View {
+    let model: ReadingListCellViewModel
+
+    init(model: ReadingListCellViewModel) {
+        self.model = model
+    }
+
+    public var body: some View {
+        VStack {
+            HStack {
+                VStack(alignment: .leading) {
+                    Text(model.title)
+                    Text(model.subtitle)
+                }
+                Spacer()
+                AsyncImage(url: model.thumbnailURL) { phase in
```

Comment:
curious if you were able to get the native `AsyncImage` working properly, last time I checked, it was still a bit glitchy

Reply:
I haven't yet investigated it, this UI code can be considered placeholder.